autowrap/DeclResolver.py

```diff
@@ -596,7 +597,8 @@ def _resolve_method_or_function(method_decl, instance_mapping, local_type_map,
         arg_type = _resolve_alias(arg_type, instance_mapping, local_type_map)
         args.append((arg_name, arg_type))
     name = method_decl.annotations.get("wrap-as", method_decl.name)
-    return clz(name, result_type, args, method_decl, instance_mapping, local_type_map)
+
+    return clz(name, method_decl.is_static, result_type, args, method_decl, instance_mapping, local_type_map)
```

Comment:
add comment what clz means?

Reply:
clz is a param of the function. There are too many instances of this kind of usage.
There should be a big PR with typing support.